content/en/docs/components/katib/reference/nas-algorithms.md

```diff
@@ -11,7 +11,7 @@ This page describes how Neural Architecture Search (NAS) algorithms work in Kati
 The algorithm follows the idea proposed in _Efficient Neural Architecture Search via Parameter Sharing_
 by Hieu Pham, Melody Y. Guan, Barret Zoph, Quoc V. Le and Jeff Dean (https://arxiv.org/abs/1802.03268)
 and _Neural Architecture Search with Reinforcement Learning_ by Barret Zoph and
-Quoc V. Le (https://arxiv.org/abs/1611.01578).
+Quoc V. Le (https://arxiv.org/abs/1611.01578) .
```

Comment:
Just for clarity... Although the link itself didn't change, without the whitespace Netlify was not rendering the link properly. Probably due to the dot in the link.

Reply:
Yes exactly the problem was the ".". adding the space solved the problem. otherwise the link was generated with the ")" and the document was not found.

Reply:
@hbelmiro  I added the screenshot with the content not loading to clarify what the issue was.

Reply:
Perfect. Thank you @varodrig.